resources/assets/coffee/_classes/forum.coffee

```diff
@@ -137,6 +139,16 @@ class @Forum
 
     true
 
+  keyboardNavigation: (e) =>
+    return if $(e.target).is('input, select, textarea') or e.target.isContentEditable or not @_postsCounter.length
```

Comment:
Move the input check to `osu_common`? Also perhaps skip using jquery to avoid creating jquery object every keypress.